src/Uno.UI.RuntimeTests/Tests/Uno_UI/Given_ResourceResolver.cs

```diff
@@ -4,27 +4,131 @@
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using Microsoft.UI.Xaml;
 using static Private.Infrastructure.TestServices;
+using Microsoft.UI.Xaml.Controls;
+using System.Threading.Tasks;
+using Uno.UI.RuntimeTests.Helpers;
+using Uno.UI.Extensions;
 
-namespace Uno.UI.RuntimeTests.Tests.Uno_UI
+namespace Uno.UI.RuntimeTests.Tests.Uno_UI;
+
+[TestClass]
+public class Given_ResourceResolver
 {
-	public class MyCustomClass
+	[TestMethod]
+	[RunsOnUIThread]
+	public void When_Resolving_String_Resources_Should_Produce_Target_Type()
 	{
-		public Duration Duration { get; set; }
+		var SUT = new Resolve_String_Resources();
+		WindowHelper.WindowContent = SUT;
+		var expected = new Duration(TimeSpan.Parse("00:00:00.250"));
+
+		Assert.AreEqual(expected, SUT.ReferencingStringDurationFromDP.Duration);
+		Assert.AreEqual(expected, SUT.ReferenceStringDurationFromProperty.Duration);
 	}
 
-	[TestClass]
-	public class Given_ResourceResolver
+	[TestMethod]
+	[RunsOnUIThread]
+	public async Task When_Resolving_FrameworkElement_Local_ThemeResource()
```

Comment:
Looks like this is failing on iOS.

Reply:
yeah, the ios intialization sequence is messing it up.
trying to fix it in .cs was not possible, for now.
i need to bring bac kthe xaml-sourcegen portial